test_drupal_quicksprint.sh

```diff
@@ -15,12 +15,23 @@ tar -C /tmp -zxf ~/tmp/drupal_sprint_package.no_docker.$(cat ./.quicksprint_rele
 # Run the standard install_ddev.sh, asserting we already have docker set up.
 cd /tmp/drupal_sprint_package && echo y | ./install_ddev.sh
 
+
 # Remove any existing sprints, then do start_sprint.sh
-cd ~/Sites/sprint && chmod -R u+w sprint-* && rm -rf sprint-* echo y | ./start_sprint.sh
+cd ~/Sites/sprint
+find ~/Sites/sprint -type d -name "sprint-2*" -exec chmod -R u+w '{}' \; -exec rm -rf '{}' \;
+echo y | ./start_sprint.sh
 
 # Run start_sprint in the (only) sprint directory
 cd ~/Sites/sprint/sprint-2* && echo y | ./start_clean.sh
 
+# Confirms web and router status.
+ddev describe | grep running || { echo "ERROR: ddev web is not running correctly?"; exit 1; }
+ddev describe | grep "DDEV ROUTER STATUS: healthy" || { echo "ERROR: ddev router status is not healthy?" ; exit 1; }
```

Comment:
Just for grins, it's probably better to use a different exit status for each item.

Also, if we're using `set -o errexit pipefail` none of this is necessary (but this is far more explicit and excellent)

Reply:
I took a stab at some "constants" for exit statuses, but I think it's wrong. I should have used either less than 128 or less than 255. Whoops. Fix for that soon.